cpp/src/io/csv/datetime_parser.cuh

```diff
@@ -140,10 +140,10 @@ __inline__ __device__ long findFirstOccurrence(const char *data, long start_idx,
  * @return returns the number of days since epoch
  */
 __inline__ __device__
-gdf_date32 parseDateFormat(const char *data, long start_idx, long end_idx, bool dayfirst) {
+int32_t parseDateFormat(const char *data, long start_idx, long end_idx, bool dayfirst) {
```

Comment:
What happened to the date type aliases?

Reply:
Jake had suggested filing an issue to have the date format code use the wrapper types directly. There are a bunch of nested calls to date conversions functions that all directly work with int32/int64 and only wrap it up at the very end. But this code is all going to be replaced with chrono timestamps. I guess I can also just leave this alone.

Reply:
Yes, this code will need to be updated to use the wrapper types, but that was outside the scope of this task. As such, it was easiest to just update to use the same corresponding native type directly such that the deprecated type definitions could be removed.

Reply:
Alternatively, I could change this to cudf::date32::value_type. Should I leave it as is, use stdint types, or use the wrapper value_type?